### PR TITLE
Updated doctree for 0.13.0 release candidate

### PR DESCRIPTION
**Pull request by Nayef211**

### CODE REVIEW

**Review by parmeet — COMMENTED** [review]
Thanks @Nayef211 for this PR. As discussed, let's wait for the tag to be created. We should have docs auto-generated for us. If not, we can land this PR.

**Comment by Nayef211** [comment]
> Thanks @Nayef211 for this PR. As discussed, let's wait for the tag to be created. We should have docs auto-generated for us. If not, we can land this PR.

Looks like the files have been autogenerated. Modified this PR to just update the doctree.

File: versions.html [hunk 1]
```diff
@@ -20,20 +20,20 @@ <h1> PyTorch Documentation </h1>
         <li class="toctree-l1">
           <a class="reference internal" href="main/">main (unstable)</a>
         </li>
+        <li class="toctree-l1">
+          <a class="reference internal" href="0.13.0/">v0.9.0 (release candidate)</a>
```
Comment (parmeet) [inline versions.html:24]: could you please update 0.9.0 to 0.13.0.

File: versions.html [hunk 2]
```diff
@@ -20,20 +20,20 @@ <h1> PyTorch Documentation </h1>
...
+        </li>
         <li class="toctree-l1">
           <a class="reference internal" href="0.12.0/">v0.12.0 (stable release)</a>
         </li>
-       <li class="toctree-l1">
+        <li class="toctree-l1">
```
Comment (parmeet) [inline versions.html:29]: not sure if it matter, but can we remove extra space in the front?

File: versions.html [hunk 3]
```diff
@@ -20,20 +20,20 @@ <h1> PyTorch Documentation </h1>
...
         <li class="toctree-l1">
-          <a class="reference internal" href="0.9.0/">v0.9.0</a>
+          <a class="reference internal" href="0.10.0/">v0.10.0</a>
         </li>
       </ul>
     </div>
     </div></div>
   </body>
-</html>
+</html>
```
Comment (parmeet) [inline versions.html:39]: looks like the newline is removed from the end?

File: versions.html (hunk 1 above)
Comment (Nayef211) [reply versions.html]: Oops thanks for the catch!

File: versions.html (hunk 2 above)
Comment (Nayef211) [reply versions.html:29]: So this is actually removing the additional space that was there before to make this line consistent with how we indent the other `<li` lines

File: versions.html (hunk 3 above)
Comment (Nayef211) [reply versions.html]: Added

**Review by parmeet — APPROVED** [review]
LGTM!

**Pull request merged**